aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -1828,18 +1828,15 @@ TEST(NewOperatorRegistrationTest, dispatchMultipleTensors) {
   }
 
   {
-    // TODO(#43908): currently this will fallthrough AutogradPrivateUse1 then call catchall kernel
-    // at AutogradCPU, while backend extenders are indeed expecting to call PrivateUse1 kernel.
-    // This confusing behavior is caused by we registering fallthrough as backend fallback for
-    // Autograd keys. Note users could always work around this by registering the same kernel to
-    // AutogradPrivateUse1 as shown below until we support it.
```

Comment:
The above issue should actually be fixed by this PR. See my comment at https://github.com/pytorch/pytorch/issues/43908#issuecomment-989133072.